harness/determined/pytorch/_data.py

```diff
@@ -95,6 +104,11 @@ def __init__(
         drop_last: bool = False,
         timeout: float = 0,
         worker_init_fn: _worker_init_fn_t = None,
+        multiprocessing_context=None,
```

Comment:
are you going to document this argument as well?

Reply:
I now see that torch does not document it, either. Maybe we could leave it undocumented. Or maybe you can steal from https://github.com/pytorch/pytorch/blob/master/torch/utils/data/dataloader.py#L335 . Something like
`A valid context object or a string specifying the start method. (default: None)`.

Reply:
We are not here to fix or alter the DataLoader from pytorch, except for exactly the changes we need, so I left this undocumented as they did.